src/routes/mainRouter.js

```diff
@@ -1,13 +1,26 @@
 const express = require("express");
-const { signUp, logIn, logOut } = require("../controllers/userController");
+const {
+  signUp,
+  logIn,
+  logOut,
+  getAdmin,
+  getUser,
+} = require("../controllers/userController");
 const signupDataValidate = require("../middlewares/signupDataValidate");
 const loginDataValidate = require("../middlewares/loginDataValidate");
 const authenticateUser = require("../middlewares/authenticateUser");
+const adminRole = require("../middlewares/adminRole");
 const router = express.Router();
 
 //user
 router.post("/signup", signupDataValidate, signUp);
-router.route("/login").post(loginDataValidate, logIn).get(authenticateUser);
+router
+  .route("/login")
+  .post(loginDataValidate, logIn)
+  .get(authenticateUser, getUser);
```

Comment:
It's confusing to me that when I send a GET request to `/login`, I get the user info that is logged in. Maybe you could change it? An alternative is `/currentuser`.